codec-redis/src/main/java/io/netty/handler/codec/redis/RedisMessagePool.java

```diff
@@ -26,19 +26,19 @@
 public interface RedisMessagePool {
 
     /**
-     * Returns {@link SimpleStringRedisMessage} for given {@code content}. Returns {@code null} it does not exist.
+     * Returns {@link SimpleStringRedisMessage} for given {@code RedisReplyKey}. Returns {@code null} it does not exist.
      */
-    SimpleStringRedisMessage getSimpleString(String content);
+    SimpleStringRedisMessage getSimpleString(FixedRedisMessagePool.RedisReplyKey key);
 
     /**
      * Returns {@link SimpleStringRedisMessage} for given {@code content}. Returns {@code null} it does not exist.
      */
     SimpleStringRedisMessage getSimpleString(ByteBuf content);
 
     /**
-     * Returns {@link ErrorRedisMessage} for given {@code content}. Returns {@code null} it does not exist.
+     * Returns {@link ErrorRedisMessage} for given {@code RedisErrorKey}. Returns {@code null} it does not exist.
      */
-    ErrorRedisMessage getError(String content);
+    ErrorRedisMessage getError(FixedRedisMessagePool.RedisErrorKey key);
```

Comment:
API Breakage

Reply:
@jjz921024 you can't add new methods to an interface as this is an API breakage.

Reply:
@normanmaurer @hyperxpro Sorry, I didn't realize that. So, I can only add methods to the impl class of this interface?
I will fixed it

Reply:
> Can you add javadocs to the two public methods ?

@normanmaurer yes, I've added javadocs to those two methods.